orbital-mechanics/Lecture7/Lecture7.ipynb

```diff
@@ -1,386 +1,425 @@
 {
```

Comment:
<p>in the first sentence, please put h within $$ </p><p> </p><p>in bullet point 1, put $$ around r.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>